src/main/scala/latis/input/HapiJsonAdapter.scala

```diff
@@ -0,0 +1,45 @@
+package latis.input
+
+import latis.model.DataType
+
+/**
+ * Adapter for HAPI JSON datasets.
+ */
+class HapiJsonAdapter(model: DataType, config: TextAdapter.Config = new TextAdapter.Config("dataMarker" -> "\"data\":\\["))
+  extends TextAdapter(model, config) {
+
+  /**
+   * Extract the data values from the given record.
+   */
+  override def extractValues(record: String): Vector[String] =
+    splitAtDelim(record).map(removeBracketsAndQuotes(_))
+
+  /**
+   * Split the given string based on the configured delimiter.
+   * The delimiter can be a regular expression.
+   * A trailing delimiter will not yield an empty string.
+   */
+  override def splitAtDelim(str: String): Vector[String] =
+    str.trim.split(config.delimiter).toVector
+
+  /**
+   * Remove all square brackets and/or quotes from the edges of a string.
+   */
+  def removeBracketsAndQuotes(str: String): String =
+    str.replaceAll("^(\\[\")|^[\\[\"]|[\"\\]]$|(\"\\])$", "")
+}
```

Comment:
These seem general purpose enough to be a part Latis3 utils. It would be a pain to move now and make a separate PR, but I'll just put that out there.